### ISSUE TITLE
Block authedmine.com

### ISSUE THREAD

**Issue by ZeroDot1** [keraf/NoCoin#67]
https://authedmine.com/lib/authedmine.min.js

**Comment by keraf** [comment]
I'm not sure it would be a good idea to block authedmine as the intention behind it is rather good. The goal of the extension is mainly to block sneaky miners and provide the user an alternative to opt-in when the site doesn't. This is exactly what authedmine does and I think adding it to the blacklist would be counter productive as miner services would just find new ways to bypass mine blockers. I guess in the future I could add a strict blacklist containing this domain but disabled by default (for users that really wish to know nothing about mining). Any opinions on that?

**Comment by Sajo811** [comment]
I agree with keraf- authedmine was created so that the domain isn't blocked in return for the user having to explicitly opt in. No use in blacklisting it.(something which malwarebytes fails to grasp)
